core/config/project_settings.cpp

```diff
@@ -1232,6 +1232,19 @@ const HashMap<StringName, ProjectSettings::AutoloadInfo> &ProjectSettings::get_a
 	return autoloads;
 }
 
+TypedArray<Dictionary> ProjectSettings::_get_autoload_list() const {
+	TypedArray<Dictionary> result;
+	for (KeyValue<StringName, AutoloadInfo> kv : autoloads) {
+		Dictionary kvData;
+		kvData["name"] = kv.value.name;
+		kvData["path"] = kv.value.path;
+		kvData["is_singleton"] = kv.value.is_singleton;
```

Comment:
This is a _really_ bad name. I know internally it is named like this, but it really should have never been in my opinion. "singleton" and "global variable" are not the same thing.
So if it needs to be exposed it needs to be called something else (`is_global_variable` can do, too).